pallets/runtime/src/asset.rs

```diff
@@ -391,6 +397,24 @@ decl_module! {
         /// * `asset_type` - the asset type.
         /// * `identifiers` - a vector of asset identifiers.
         /// * `funding_round` - name of the funding round
+        ///
+        /// # Weight
+        /// `40_000 + 1_000 * identifiers.len()`
+        #[weight = FunctionOf(
+            |args: (
```

Comment:
Using argument position (`args.5.len()`) is hard to see what argument are you using.
We could use something like:
```
        #[weight = FunctionOf(
            |(_, _, _, _, _, identifiers, _):( &TokenName, &Ticker, &T::Balance, &bool, &AssetType, &Vec<(IdentifierType, AssetIdentifier)>, &Option<FundingRoundName>
            )| {
                40_000 + 1_000 * u32::try_from(identifiers.len()).unwrap_or_default()
            },
```